ruby/secret_agents.rb

```diff
@@ -0,0 +1,80 @@
+#encrypt 
+#def the method and ask for the string
+#define alphabet variable
+#find the index of the string and added 1 to the index value
+#this will allow us to find the next value in the alphabet
+
+def encrypt(password)
+	def ascend (letter)
```

Comment:
defining methods inside methods is VERY DANGEROUS.  Refactor this with the methods defined individually.  There is nothing wrong with calling a method inside a method but they need to be defined separately.